catalyst-gateway/bin/src/cardano/mod.rs

```diff
@@ -312,7 +301,7 @@ fn sync_subchain(
                     }
 
                     if chain_update.tip && !set_follower_live_first_reached_tip() {
-                        let _ = event_sender.send(event::ChainIndexerEvent::SyncLiveChainCompleted);
+                        metrics_updater::reached_tip(true, true);
```

Comment:
no no, we should update only for live tip, but this seems updating `immutable` metric